lib/logitech_receiver/settings_templates.py

```diff
@@ -195,6 +202,26 @@ def _feature_k375s_fn_swap():
 					label=_FN_SWAP[1], description=_FN_SWAP[2],
 					device_kind=(_DK.keyboard,))
 
+def __feature_disable_keys(k, value):
+	return feature_toggle(k[0], _F.KEYBOARD_DISABLE_KEYS,
+					true_value=value, mask=value,
+					label=k[1], description=k[2],
+					read_function_id=0x10,
+					write_function_id=0x20,
+					device_kind=(_DK.keyboard,))
+
+def _feature_disable_caps_lock():
+	return __feature_disable_keys(_DISABLE_CAPS_LOCK, 0x01)
+
+def _feature_disable_scroll_lock():
+	return __feature_disable_keys(_DISABLE_SCROLL_LOCK, 0x04)
+
+def _feature_disable_insert():
+	return __feature_disable_keys(_DISABLE_INSERT, 0x08)
+
+def _feature_disable_win():
+	return __feature_disable_keys(_DISABLE_WIN, 0x10)
+
 # FIXME: This will enable all supported backlight settings, we should allow the users to select which settings they want to enable.
```

Comment:
I worry that these are somehow specific to the 400+ and other keyboards have a different mapping.
@FFY00 Can we get documentation on this feature (KEYBOARD_DISABLE_KEYS)?

Reply:
Even if these bit masks are general, I very strongly suspect that different keyboard have different keys that can be disabled.  It appears to me that this implementation will have the consequence that very keyboard with this feature will get all these keys in the UI but the keyboard may only implement some of them.  I think that command 0x00 will return a bit mask saying which keys can be disabled.  Hopefully there will be documentation available fairly soon to determine whether this is true or not, but for now I think this commit should not be mergerd.

Reply:
I agree with you. In fact, your guess is right: 0x00 returns 0x1D, which is precisely 0x01 + 0x04 + 0x08 + 0x10. Hence, the correct implementation should run this in order to decide which keys should be shown in the UI, but this would require knowing the meaning of other codes (0x02, 0x11, ...) that may be present on other models that support this feature.

Reply:
have a new PR that includes the possibility that a setting should not be active even if the device has the feature.  See feature_map_choices_dynamic in setings_template.py in PR #810.  I don't think that having an auxiliary function is necessary for this to work - just do a feature request for the value, check it for the correct bit, and if not set return None.